### ISSUE TITLE
Code modernization

### ISSUE THREAD

**Issue by maciek134** [eduardoboucas/staticman#249]
I took a look at the source code while working on https://github.com/eduardoboucas/staticman/issues/243#issuecomment-450472431 and I see that some of it looks old - creating classes with `function` and `.prototype...`, promises instead of `async/await`. If I rewrite the codebase to use more modern JS features will a PR with it be accepted?

A bit off topic, but why StandardJS? The lack of semicolons gives me shivers whenever I look at the code ;p

**Comment by alexwaibel** [comment]
Yeah, this code is using old standards and styles. At some point, I'd like to move to the Airbnb style guide and convert the whole thing to ES6 but that's a low priority task for the time being. I do find it important to modernize our style as the current standards are very difficult to read in my opinion and will deter open source contributions if we don't fix it.
